src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -1064,6 +1076,8 @@ private void SetAffinity(Process process, int cores, bool addPerformanceHint)
                 return;
             }
 
+            _serverStore.ConcurrentBackupsCounter.SetMaxNumberOfConcurrentBackups(cores);
```

Comment:
This should be applied after `IgnoreProcessorAffinityChanges` check

Reply:
Shouldn't we just use LicenseChanged event?

Reply:
Done. SetAffinity is called as part of the license changed event

Reply:
And moved it to be part of the LicenseChanged event.